sql-statements/sql-statement-admin-checksum-table.md

```diff
@@ -0,0 +1,52 @@
+---
+title: ADMIN CHECKSUM TABLE | TiDB SQL Statement Reference
+summary: An overview of the usage of ADMIN for the TiDB database.
+category: reference
+---
+
+# ADMIN CHECKSUM TABLE
+
+The `ADMIN CHECKSUM TABLE` statement calculates a CRC64 checksum for the data and indexes of a table. This statement is used by programs such as TiDB-Lightning to ensure that import operations have completed successfully.
```

Comment:
should we mention that the result is different from MySQL?

Reply:
I would normally put that under MySQL Compatibility, but this statement already says it is an extension.